UI/Pages/SchoolLedTutoring.cshtml

```diff
@@ -1,9 +1,63 @@
 ﻿@page
 @{
-    Layout = "_Layout";
-    ViewData["Title"] = "SchoolLedTutoring - Compare National Tutoring Options";
+  Layout = "_Layout";
+  ViewData["Title"] = "SchoolLedTutoring - Compare National Tutoring Options";
 }
-
-<h1 class="govuk-heading-xl">School-led tutoring</h1>
-
-<p class="govuk-body">School-led tutoring content</p>
\ No newline at end of file
+<div class="govuk-grid-row">
```

Comment:
Last updated and  "how much funding will I get link" missing on the right hand side of doc.